scrapli_community/huawei/vrp/huawei_vrp.py

```diff
@@ -57,9 +57,7 @@
         "sync_on_close": default_sync_on_close,
         "async_on_close": default_async_on_close,
         "failed_when_contains": [
-            "Error: Unrecognized command found at '^' position.",
-            "Error: Wrong parameter found at '^' position.",
-            "Error:Incomplete command found at '^' position.",
```

Comment:
missing a space between "Error:" and "Incomplete" maybe?

Reply:
This is actually one of the things that differ from one VRP version to the other. 
Some versions/devices output the error with a space between the ":" and the actual message, and some without.

Reply:
One of the main reasons why I decided to only match `Error:` instead of the full error messages.

Reply:
lol im an idiot :) was looking at the old side vs the new side.... carry on sir :D lol

Reply:
Haha, no worries! :)